packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -239,6 +265,159 @@ export default class StatsAnalyzer extends EventsScope {
     });
   }
 
+  /**
+   * parse the audio
+   * @param {String} result
+   * @param {boolean} type
+   * @returns {void}
+   */
+  parseAudioSource(result, type) {
+    if (!result) {
+      return;
+    }
+
+    if (type === STATS.AUDIO_CORRELATE) {
+      this.statsResults[type].send.audioLevel = result.audioLevel;
+      this.statsResults[type].send.totalAudioEnergy = result.totalAudioEnergy;
+    }
+  }
+
+  /**
+   * compares current and previous stats to check if packets are not sent
+   *
+   * @private
+   * @memberof StatsAnalyzer
+   * @returns {void}
+   */
+  compareLastStatsResult() {
+    if (this.lastStatsResults !== null) {
+      // compare audio stats sent
+      let mediaType = STATS.AUDIO_CORRELATE;
+      let currentStats = null;
+      let previousStats = null;
+
+      if (this.meetingMediaStatus.expected.sendAudio) {
+        currentStats = this.statsResults[mediaType].send;
+        previousStats = this.lastStatsResults[mediaType].send;
+
+        if (currentStats.totalPacketsSent === previousStats.totalPacketsSent || currentStats.totalPacketsSent === 0) {
+          LoggerProxy.logger.log(`StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} RTP packets sent`);
```

Comment:
Do all of these need to be at `log` level? `info` or `debug` might reduce the noise, but if you think the end user really needs to know about them, we can leave at log.